modules/tinymce/src/core/main/ts/api/html/SaxParser.ts

```diff
@@ -206,16 +208,10 @@ function SaxParser(settings?: SaxParserSettings, schema = Schema()): SaxParser {
   const end = settings.end ? settings.end : noop;
   const pi = settings.pi ? settings.pi : noop;
   const doctype = settings.doctype ? settings.doctype : noop;
+  const dataUri = settings.dataUri ? settings.dataUri : buildBase64DataUri;
 
-  /**
-   * Parses the specified HTML string and executes the callbacks for each item it finds.
-   *
-   * @example
-   * SaxParser({...}).parse('<b>text</b>');
-   * @method parse
-   * @param {String} html Html string to sax parse.
-   */
-  const parse = (html: string, format: ParserFormat = 'html') => {
+  const parseHtml = (base64Extract: Base64Extract, format: ParserFormat = 'html') => {
```

Comment:
This name isn't really appropriate, as this can parse HTML and XHTML (and technically it works as a lenient XML parser).

Reply:
Name suggestions? parseMarkup, doParse.

Reply:
`internalParse` maybe? otherwise either of those look good to me.